apps/ios/Shared/Views/UserSettings/NetworkAndServers/AdvancedNetworkSettings.swift

```diff
@@ -28,19 +28,20 @@ struct AdvancedNetworkSettings: View {
     @EnvironmentObject var theme: AppTheme
     @AppStorage(DEFAULT_DEVELOPER_TOOLS) private var developerTools = false
     @AppStorage(DEFAULT_SHOW_SENT_VIA_RPOXY) private var showSentViaProxy = false
-    @State private var netCfg = NetCfg.defaults
-    @State private var currentNetCfg = NetCfg.defaults
     @State private var cfgLoaded = false
     @State private var enableKeepAlive = true
     @State private var keepAliveOpts = KeepAliveOpts.defaults
     @State private var showSettingsAlert: NetworkSettingsAlert?
     @State private var onionHosts: OnionHosts = .no
     @State private var showSaveDialog = false
-    @State private var netProxy = networkProxyDefault.get()
-    @State private var currentNetProxy = networkProxyDefault.get()
     @State private var useNetProxy = false
     @State private var netProxyAuth = false
-
+    @Binding public var currentNetCfg: NetCfg
+    @Binding public var netCfg: NetCfg
+    @Binding public var currentNetProxy: NetworkProxy
+    @Binding public var netProxy: NetworkProxy
```

Comment:
instead of passing bindings to view it should be possible to now use them directly from `@EnvironmentObject var ss: SaveableSettings`, same as in NetworkAndServers.